CODEOWNERS

```
@@ -12,6 +12,6 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @MrAlias @XSAM @dashpole @pellared @dmathieu
+* @MrAlias @XSAM @dashpole @pellared @dmathieu @flc1125
 
-CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu
+CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu @flc1125
```

```suggestion
CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu
```

Only maintainers own the CODEOWNERS file :)